x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/ModelUploader.java

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
This class is just to wrap the `ModelLoaderUtils` APIs so that I can do dependency injection to test code in the Transport class.